### PR TITLE
Add Taack PLM Workbench

### PR DESCRIPTION
**Pull request by Taack**
Add Taack PLM to git modules file.

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -354,6 +354,9 @@
 	path = toSketch
 	url = https://github.com/KeithSloan/toSketch.git
 	branch = main
+[submodule "taack-plm-freecad"]
+	path = taack-plm-freecad
+	url = https://github.com/Taack/taack-plm-freecad.git
```
Comment (chennes) [inline .gitmodules:359]:
```suggestion
[submodule "taack-plm-freecad"]
	path = taack-plm-freecad
	url = https://github.com/Taack/taack-plm-freecad.git
	branch = main
```

Comment (chennes) [reply .gitmodules:359]: Don't forget the branch line (@luzpaz IMO we should add a `branch = ` line to all of them, so it's obvious that you should have it. It defaults to `master` for historical reasons, but most new devs use `main`)

**Review by chennes — CHANGES_REQUESTED** [review]
Needs to specify the branch, but otherwise this is good to merge.

**Comment by Taack** [comment]
Just to clarify, I prefer branch **main** than a precise **tag**. No more breaking change should be pushed in the Workbench repo. Thanks!

**Review by chennes — APPROVED** [review]

**Pull request merged**